src/components/Examples.tsx

```diff
@@ -98,46 +107,75 @@ const Header = styled.div`
   }
 `;
 
+const ExampleContainer = styled.div`
+  display: grid;
+  grid-template-columns: repeat(4, minmax(100px, 320px));
+  grid-gap: 1rem;
+  a, .full-height{
+    height: 100%;
+  }
+  
+`;
+
+const Buttons = styled.div`
+  display: grid;
+  grid-template-columns: repeat(2, 1fr);
+  grid-gap: 10px;
+  width: 100%;
+  
+  align-items: flex-start;
+  
+  button{
+    width: 100%;
+  }
+`
+
+
 const Example = styled.div`
-  display: flex;
+  display: grid;
+  grid-template-rows: 80px auto 1fr auto;
+  grid-gap: 1rem;
   flex-direction: column;
-  align-items: flex-start;
-  justify-content: center;
-  text-align: left;
-  min-height: 250px;
-  max-width: 325px;
-  margin-right: 0.5rem;
-  margin-left: 0.5rem;
-  margin-top: 1rem;
-  //border: 2px solid rgb(240, 240, 240);
+  align-items: center;
+  justify-content: flex-start;
+  height: 100%;
   border-radius: 10px;
   padding: 2rem;
   cursor: pointer;
```

Comment:
Given that clicking on the example actually doesn't do anything, I think it's best to not confuse users and remove this now:

```suggestion
```